### ISSUE TITLE
ORKTextChoiceOther.choice Seemingly not working with iOS 14

### ISSUE THREAD

**Issue by scdi** [ResearchKit/ResearchKit#1406]
It seems that the ORKTextChoiceOther.choice is working in iOS 13 but NOT with iOS 14. 
A click in the TextBox that opened when the other choice is selected closes the TextBox rather than allowing the user to enter a text.
![Screen Recording 2020-08-25 at 10 08 43 AM](https://user-images.githubusercontent.com/13545391/91185394-c49b3400-e6bb-11ea-8d66-c4e73f85968a.gif)

**Comment by zinn-claus** [comment]
I can confirm this bug for iOS 14 as well. It also happens when textViewStartsHidden is set to false. This is very annoying. Since the issue was raised a couple of months ago, what are the chances that it will be fixed soon?

**Comment by smclelland** [comment]
Yes, can confirm that I'm seeing the same thing. Had to disable for now.

**Comment by marius-serban** [comment]
There's a fix on a branch:  #1429. If you want, you can merge that into your fork.
